man/org.freedesktop.resolve1.xml

```diff
@@ -134,7 +134,7 @@ node /org/freedesktop/resolve1 {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
       readonly a(isb) Domains = [...];
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
-      readonly (tt) TransactionStatistics = ...;
+      readonly (tttttt) TransactionStatistics = ...;
```

Comment:
unfortunately dbus is not extensible this way: old clients would misprase the changed property. hence we cannot change existing properties, we can only add new properties.

Hence maybe add a new property "TransactionFailureStatistics" or so, where you put the new fields.